### PR TITLE
Lightwave

### LINKED ISSUE #1796: Wavelite - make Wave integratable within arbitrary python web frameworks (Django, Flask, FastAPI etc.)
Make the integration very similar to what https://nitro.h2o.ai/intro/ did.

### PR DESCRIPTION
**Pull request by mturoci**
Closes #1796

### CODE REVIEW

File: py/h2o_wavelite/README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# H2O Wavelite
+
+H2O Wavelite is a lighter version of [H2O Wave](https://wave.h2o.ai/) consisting of python driver only, no Golang server. This allows for seamless integration into popular existing python web frameworks like Django, Flask, FastAPI, Starlette etc.
```
Comment (lo5) [inline py/h2o_wavelite/README.md:3]:
```suggestion
H2O Wave Lite is a lightweight, pure-Python version of [H2O Wave](https://wave.h2o.ai/) that can be embedded in popular web frameworks like  Django, Flask, FastAPI, Starlette, etc. 

In other words, H2O Wave Lite works without the Wave server.
```

File: py/h2o_wavelite/README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+The integration consists of 2 steps:
+
+* Serve Wavelite's static web dir at the route of your choice.
```
Comment (lo5) [inline py/h2o_wavelite/README.md:7]:
```suggestion
* Add Wave's web assets directory to your framework's static file handler.
```

File: py/h2o_wavelite/README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+* Set up a WebSocket connection and hook `wave_serve` callback function to control the UI.
```
Comment (lo5) [inline py/h2o_wavelite/README.md:8]:
```suggestion
* Add a WebSocket handler, and use `wave_serve()` to connect connect Wave to your web UI.
```

File: py/h2o_wavelite/README.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+Done. You can render UI elements with nothing but Python. Wavelite aims to be as simplistic as possible and only provides:
```
Comment (lo5) [inline py/h2o_wavelite/README.md:10]:
```suggestion
That's it. You can now render UI elements using pure Python. Wave Lite aims to be as minimal as possible and only provides:
```

**Review by lo5 — CHANGES_REQUESTED** [review]
I'd prefer "Wave Lite" instead of "Wavelite" if that's the name we're going for.
We can also brainstorm other names (lightwave?).

**Comment by lo5** [comment]
Overall, great PR. I like how minimal the changes are. 

What about the file upload/download routines from Wave - those won't function properly in the light version, correct?

**Comment by mturoci** [comment]
>What about the file upload/download routines from Wave - those won't function properly in the light version, correct?

They are not part of the python API at all meaning the type system should take care of it. Will make this clear in the README though.

**Comment by mturoci** [comment]
>We can also brainstorm other names (lightwave?)

Definitely, created a poll in Slack to get broader ideas. Wavelite was just the first thing that came to my mind, not final for sure.

**Pull request merged**